apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/ServerExecutor.java

```diff
@@ -63,20 +62,18 @@ public static synchronized boolean startServer(String dbUrl, String dbUsername,
     String targetPath = "target";
     String workDirectory = targetPath + "/server-work";
     String warArtifactLocation = targetPath + "/" + TEST_WEBAPP_DIRECTORY;
-    String serverPortsFile = workDirectory + "/server-ports.properties";
     // These two files are copied into server-work during build time by maven for convenience
     // from: bfd-server/dev/ssl-stores
     String keyStore = workDirectory + "/server-keystore.pfx";
     String trustStore = workDirectory + "/server-truststore.pfx";
 
     // Validate the paths and properties needed to run the server war exist
     if (!validateRequiredServerSetup(
-        targetPath, workDirectory, warArtifactLocation, serverPortsFile, keyStore, trustStore)) {
+        targetPath, workDirectory, warArtifactLocation, keyStore, trustStore)) {
       return false;
     }
 
-    String portFileContents = Files.readString(Path.of(serverPortsFile)).trim();
-    String serverPort = portFileContents.substring(portFileContents.indexOf('=') + 1);
+    String serverPort = "0";
     LOGGER.info("Configured server to run on HTTPS port {}.", serverPort);
```

Comment:
We should move this logger info down to line 113, otherwise we don't get any useful info from it.

Reply:
ok will do that